docker/docker-compose-dev.yaml

```diff
@@ -44,10 +44,10 @@ services:
     environment:
       WREN_AI_SERVICE_PORT: ${WREN_AI_SERVICE_PORT}
       WREN_UI_ENDPOINT: ${WREN_UI_ENDPOINT}
-      GENERATION_MODEL: ${GENERATION_MODEL}
-      OPENAI_API_KEY: ${OPENAI_API_KEY}
-      AZURE_CHAT_KEY: ${AZURE_CHAT_KEY}
-      AZURE_EMBED_KEY: ${AZURE_EMBED_KEY}
+      LLM_OPENAI_API_KEY: ${LLM_OPENAI_API_KEY}
```

Comment:
@cyyeh  could you help me understand the differences between `LLM_OPENAI_API_KEY` and `EMBEDDER_OPENAI_API_KEY` and the usage of it?

Reply:
@onlyjackfrost  After this PR, all of the configurations of LLM and EMBEDDER are separated. So users can flexibly choose what LLM and Embedder they need using different providers. For example. LLM using OpenAI, Embedder using Fireworks.ai(OpenAI api compatible).

Reply:
Thanks for sharing